src/modules/bookmark/utils.js

```diff
@@ -0,0 +1,73 @@
+import { tokenKeys, tokenMap } from '@token/configuration/tokens';
```

Comment:
Can these be split into separate utilities?

Reply:
as i said before if make it as directory they need to be split if its just few of them keep it
don't wast the time for such things we need to finish movement as soon as posible if we have extra time we will spend on it later on this sprint